package.json

```diff
@@ -4,30 +4,35 @@
   "description": "An Application Architecture Using Backbone.js",
   "repository": "git://github.com/chaplinjs/chaplin.git",
   "dependencies": {
+    "babel-register": "6.18.0",
     "backbone": "~1.3.3",
     "underscore": "~1.8.3"
   },
   "devDependencies": {
+    "babel-plugin-transform-es2015-modules-commonjs": "6.18.0",
     "backbone.nativeview": "~0.3.3",
     "chai": "3.5.x",
     "coffee-coverage": "1.0.1",
-    "coffee-script": "1.10.x",
-    "coffeeify": "~2.0.1",
+    "coffee-script": "1.11.x",
     "grunt": "~0.4.5",
-    "grunt-browserify": "~4.0.1",
     "grunt-cli": "~0.1.13",
     "grunt-coffeelint": "~0.0.15",
+    "grunt-contrib-clean": "1.0.0",
+    "grunt-contrib-coffee": "florian-r/grunt-contrib-coffee",
```

Comment:
This is WIP pending a new release of `grunt-contrib-coffee`. The current version use an old version of `coffee-script`.